### PR TITLE
[WIP] Feature button badge

### PR DESCRIPTION
**Pull request by brianjd**
Still a WIP, but submitting pull for tracking and feedback.

image

- [x] Android Implementation
- [ ] Sort out icon sizing
- [ ] Remove built-in theme styling, allow for direct colour assignment
- [ ] iOS Implementation

### CODE REVIEW

File: android/app/src/main/java/com/reactnativenavigation/layouts/BottomTabsLayout.java [hunk 1]
```diff
@@ -182,6 +182,13 @@ public void setTitleBarLeftButton(String screenInstanceId, String navigatorEvent
     }
 
     @Override
+
+    public void setTitleBarButtonBadgeByIndex(String screenInstanceId, Integer index, Integer badge) {
+        for (int i = 0; i < bottomTabs.getItemsCount(); i++) {
```
Comment (brianjd) [inline android/app/src/main/java/com/reactnativenavigation/layouts/BottomTabsLayout.java]: Looks like I screwed up the merge 🙁

Comment (brianjd) [inline android/app/src/main/java/com/reactnativenavigation/layouts/BottomTabsLayout.java]: Looks like I screwed up the merge 🙁

**Comment by varungupta85** [comment]
hi @brianjd , are you still working on this PR? If it is ready for Android, could you please submit it for merge into Android. I think this feature is more needed in Android where single screen layout is more standard as compared to iOS where a tab based layout is more standard. Thanks!

**Comment by brianjd** [comment]
@varungupta85 I have been really busy with my own projects recently.  I will jump back into this later this week.

**Comment by varungupta85** [comment]
@brianjd that would be great, thanks. Please let me know if you want me to test it or help out with something else.

**Pull request closed without merging**